provider/oci/networking.go

```diff
@@ -185,12 +185,12 @@ func (e *Environ) allSecurityLists(controllerUUID, modelUUID string, vcnid *stri
 	}
 	for _, val := range response.Items {
 		tag, ok := val.FreeformTags[tags.JujuController]
-		if !ok || tag == controllerUUID {
```

Comment:
This is as far as we got in review today.
I think the line here seems to indicate a missed test, did it get added?

Reply:
not yet, there will be a few more tests getting added and some changed after 5/7 merges.